akka-actor-typed-tests/src/test/scala/akka/actor/typed/internal/adpater/ActorAdapterSpec.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.internal.adpater
+
+import scala.concurrent.duration.DurationInt
+import org.scalatest.wordspec.AnyWordSpecLike
+import akka.actor.testkit.typed.scaladsl._
+import akka.actor.typed._
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.adapter._
+
+class ActorAdapterSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike {
+
+  sealed trait Command
+  case object Begin extends Command
+  case object End extends Command
+  case object Timeout extends Command
+
+  sealed trait Event
+  case object Ended extends Event
+
+  val interval = 1.second
```

Comment:
this val seems not to be in use

Reply:
I think the build will fail because we have a val not in use.

Reply:
you are correct, somehow I missed then - even after running the validator.